src/java.base/share/classes/javax/crypto/CipherOutputStream.java

```diff
@@ -100,14 +102,14 @@ private void ensureCapacity(int inLen) {
 
     /**
      *
-     * Constructs a CipherOutputStream from an OutputStream and a
-     * Cipher.
+     * Constructs a <code>CipherOutputStream</code> from an
+     * <code>OutputStream</code> and a <code>Cipher</code>.
      * <br>Note: if the specified output stream or cipher is
-     * null, a NullPointerException may be thrown later when
+     * <code>null</code>, a NullPointerException may be thrown later when
```

Comment:
Add {@code } to NullPointerException also

Reply:
using `{@code }`